usual/tls/tls.c

```diff
@@ -742,4 +742,85 @@ tls_close(struct tls *ctx)
 	return (rv);
 }
 
+static bool
+tls_mems_equal(char *mem1, char *mem2, size_t len1, size_t len2)
+{
+	if (len1 != len2)
+		return false;
+	if (mem1 && mem2 && memcmp(mem1, mem2, len1) != 0)
+		return false;
+	return true;
+}
+
+static bool
+keypair_equal(struct tls_keypair *tkp1, struct tls_keypair *tkp2)
+{
+	if (!strcmpeq(tkp1->cert_file, tkp2->cert_file))
+		return false;
+	if (!tls_mems_equal(tkp1->cert_mem, tkp2->cert_mem, tkp1->cert_len, tkp2->cert_len))
+		return false;
+	if (!strcmpeq(tkp1->key_file, tkp2->key_file))
+		return false;
+	if (!tls_mems_equal(tkp1->key_mem, tkp2->key_mem, tkp1->key_len, tkp2->key_len))
+		return false;
+	return true;
+}
+
+bool
+keypair_list_equal(struct tls_keypair *tkp1, struct tls_keypair *tkp2)
```

Comment:
nit: lets prefix these functions with `tls_` for consistency with the rest of the functions.

Reply:
@eulerto felt that the functions should not have the prefix because the struct name is just keypair but my feeling was same as yours. I'm happy to go either way.

Reply:
I think you misread/misremembered his suggestion:
> I suggest that you rename `tls_config_keypairs_equal` to `tls_keypair_equal`.

Reply:
You are right that I misread his suggestion. Thanks for the heads-up.

Reply:
[426e6d3](https://github.com/libusual/libusual/pull/57/commits/426e6d3a76acfbe320c5d49400f17b70f48dc41d)

Thank you guys for your patience. I think this resolves keypair list check naming.